doEditPathology.php

```diff
@@ -10,8 +10,8 @@
 EOF;
 
     $ret = pg_query_params($db, $sql,
-        [$_POST["name"],
-        $_POST["id"]]);
+        [$_POST["pnom"],
+        $_POST["pprénom"], $_POST["pnomDeFamille"],$_POST["pAdressePermanente"],$_POST["pAdresseVillage"],$_POST["telephone"]]);
```

Comment:
please lets use ```pprenom```, in order to be consistent, since we said that we wont use accent anywhere...